library/src/main/java/org/jellyfin/apiclient/model/dto/BaseItemDto.java

```diff
@@ -1026,15 +1026,27 @@ public final void setParentId(String value)
 	 <value>The type.</value>
 	*/
 	private String Type;
+
+	@Deprecated
 	public final String getType()
 	{
 		return Type;
 	}
+
+	@Deprecated
 	public final void setType(String value)
 	{
 		Type = value;
 	}
 
+	public final void setEBaseItemType(EBaseItemType Type) {
+		this.Type = Type.name();
+	}
+
+	public final EBaseItemType getEBaseItemType() {
+		return EBaseItemType.valueOf(Type);
+	}
+
```

Comment:
Not a fan of the `E` prefix for this variable, @thornbill what do you think?

Reply:
I agree. I’m not a fan of using prefixes to indicate an object type.

Reply:
Okay, changed :)